packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/description/ImageDescription.java

```diff
@@ -153,6 +157,7 @@ public ImageDescription build() {
             imageDescription.idProvider = Objects.requireNonNull(this.idProvider);
             imageDescription.labelProvider = Objects.requireNonNull(this.labelProvider);
             imageDescription.iconURLProvider = Objects.requireNonNull(this.iconURLProvider);
+            imageDescription.isReadOnlyProvider = Objects.requireNonNull(this.isReadOnlyProvider);
```

Comment:
Same kind of remark as above, setting `imageDescription.isReadOnlyProvide` explicitly to `vm -> false` here would be clearer IMO.

Reply:
It's mandatory for the builder to declare a `isReadOnlyProvider` function, I initialized it to `variableManager -> true` for all the widgets read-only by nature.